lib-ng/scanners/base.js

```diff
@@ -105,10 +106,30 @@ var BaseScanner = function (beaconMapper, beaconRegionMapper, beaconRegionMonito
 
     this.startScanningAfterBinding = function () {
         var knownBeacons = knownBeaconService.getKnownBeacons(self.networkId);
-        if (!_.isEmpty(knownBeacons)) {
-            startScanningOfKnownBeacons(knownBeacons);
-        }
+        var realBeacons = knownBeacons.filter(knownBeacon => !knownBeacon.get('is_geofence'));
+        var geofenceBeacons = knownBeacons.filter(knownBeacon => knownBeacon.get('is_geofence'));
+        startScanningOfKnownBeacons(realBeacons);
         self.addAllEventListeners();
+        self.startScanningGeofences(geofenceBeacons);
+    };
+
+    this.startScanningGeofences = function(geofenceBeacons) {
+        // fallback for locations who doesn't have physical-BLE-Beacons
+        if (geofenceBeacons.length === 0) {
+            return;
+        }
+        const geofenceLib = require('./../scanners/geofence');
+        geofenceLib.init();
+        geofenceLib.stopMonitoring();
+        geofenceBeacons.forEach((beacon) => {
+            const identifier = `${beacon.get('beacon_id')}|${beacon.get('UUID')}|${beacon.get('major')}|${beacon.get('minor')}`;
+            geofenceLib.startMonitoring({
+                identifier,
+                latitude: beacon.get('latitude'),
+                longitude: beacon.get('longitude'),
+                radius: 100,
```

Comment:
Is it an option to make it configurable in the Sensimity Dashboard? Maybe we can create a new ticket in JIRA for it?